src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/VideoLinkMigrationController.kt

```diff
@@ -36,4 +38,26 @@ class VideoLinkMigrationController(
     @Parameter(description = "The video link booking ID to return data for.", required = true)
     @PathVariable videoBookingId: Long,
   ) = videoLinkMigrationService.getVideoLinkBookingToMigrate(videoBookingId)
+
+  @PutMapping(
+    path = ["/video-link-bookings"],
+    produces = [MediaType.APPLICATION_JSON_VALUE],
+  )
+  @ResponseStatus(HttpStatus.CREATED)
+  @Operation(
+    summary = "Trigger the process which will emit domain events for all bookings with a main hearing start time that is after the fromDate",
+    description = """
+      Requires role: ROLE_BOOK_A_VIDEO_LINK_ADMIN
+      Triggers the start of the data migration process for video link bookings
+      """,
+  )
+  @PreAuthorize("hasAnyRole('BOOK_A_VIDEO_LINK_ADMIN')")
+  fun migrateVideoLinkBookings(
+    @Parameter(description = "Earliest date for the main hearing time", required = true, example = "2023-10-01")
```

Comment:
Are these meant to be request params?  i.e. do they need the `@RequestParam` annotations?